docs/in_production.rst

```diff
@@ -40,11 +40,14 @@ Recommended environment
 Deploying Pinot
 ---------------
 
-Direct deployment of Pinot
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+In general, when deploying Pinot services, it is best to adhere to a specific ordering in which the various components should be deployed. This deployment order is recommended incase of the scenario that there might be protocol or other significant differences, the deployments go out in a predictable order in which failure  due to these changes can be avoided.
```

Comment:
The requirement section is very vague. Either expand it or take it out altogether (but we need to add zk somewhere I suppose). In the recommended section, pls add shared storage env mounted across controllers. If this storage is not shared, things will not work. Altnerativelyu, pinotfs can be used. (and ou can point to a section for that). Not clear what the load baancers are needed for. Queries to broker or rest to controlers or both.

Reply:
"in case", not "incase" right?

Reply:
I don't think that is the scope of this section. The way this was presented to me is that I need to describe how to deploy Pinot in terms of a version upgrade. 

Everything you are describing sounds like it should be part of an architecture section for how to implement Pinot

Reply:
incase -> in case

Reply:
Can you update the requirments section to be not as vague as it is now? A paragraph would do. Here is a suggestion. We can iterate and modify later, but let us have something in place now. Feel free to modify/add to as you please.

You will need the following in order to run pinot in production:
* Hardware for controller/broker/servers as per your load
* Working installation of Zookeeper that Pinot can use. We recommend setting aside a path within zookpeer and including that path in pinot.controller.zkStr. Pinot will create its own cluster under this path (cluster name decided by pinot.controller.helixClusterName)
* Shared storage mounted on controllers (if you plan to have multiple controllers for the same cluster). Alternatively, an implementation of PinotFS that the Pinot hosts have access to.
* HTTP load balancers for spraying queries across brokers (or other mechanism to balance queries)
* HTTP load balancers for spraying controller requests (e.g. segment push, or other controller APIs) or other mechanisms for distribution of these requests.

Reply:
Can we add this text under line 26 and remove the "Recommended environemnt" segction? thanks

Reply:
Done